part-spec/storage_memory.json

```diff
@@ -176,6 +226,186 @@
+                    "type": "number"
+                },
+                "bootBlockSize": {
+                    "description": "size of the eeprom boot block",
+                    "comment": "units of bytes (B)",
+                    "$ref": "definitions.json#/unit"
+                },
+                "interface": {
+                    "description": "interface of eeprom to host",
+                    "type": "string",
+                    "examples": [
+                        "spi",
+                        "i2c",
+                        "serial 1-wire"
+                    ]
+                },
+                "clockFrequency": {
+                    "description": "eeprom clock frequency",
+                    "comment": "units of Hertz (Hz)",
+                    "$ref": "definitions.json#/unit"
+                },
+                "accessTime": {
+                    "description": "time to access the eeprom",
+                    "comment": "units of seconds (s)",
+                    "$ref": "definitions.json#/unit"
+                },
+                "endurance": {
+                    "description": "time in years a bit in the eeprom can retain its data state ",
+                    "type": "number"
+                },
+                "dataRetention": {
+                    "description": "maximum number of read and write cycle the part can support",
+                    "type": "number"
+                },
+                "currentConsumption": {
+                    "description": "current consumption of a device",
+                    "comment": "units of amps",
+                    "$ref": "definitions.json#/currentConsumption"
+                },
+                "package": {
+                    "description": "component's package size and description",
+                    "$ref": "definitions.json#/package"
+                },
+                "pins": {
+                    "description": "array of pin objects with associated properties",
+                    "type": "array",
+                    "items": {
+                        "$ref": "definitions.json#/pinSpec"
+                    }
+                }
+            }
+        },
+        {
+            "title": "flash memory",
+            "type": "object",
+            "required": [
+                "componentID",
+                "capacity"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "capacity": {
+                    "description": "capacity/density of flash memory",
+                    "comment": "units of bits",
+                    "$ref": "definitions.json#/unit"
+                },
+                "pageSize": {
+                    "description": "page size of flash memory",
+                    "comment": "units of bytes (B)",
```

Comment:
same bits vs bytes comment. Why is the page size in bytes, but the capacity in bits?

Reply:
changed to bits